automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoConfig.scala

```diff
@@ -18,4 +22,27 @@ object LeonardoConfig extends CommonConfig {
 
   // for NotebooksWhitelisted
   object Users extends CommonUsers
+
+  // Hack: FlatSpec is mixed in to adhere to BillingFixtures self-type. This object does not actually define any specs.
+  object BillingProject extends FreeSpec with BillingFixtures {
+    val ronCreds: Credentials = Users.NotebooksWhitelisted.getUserCredential("ron")
+    val hermioneCreds: Credentials = Users.NotebooksWhitelisted.getUserCredential("hermione")
+
+    val ronAuthToken = UserAuthToken(ronCreds, AuthTokenScopes.userLoginScopes)
+    val hermioneAuthToken = UserAuthToken(hermioneCreds, AuthTokenScopes.userLoginScopes)
+
+    // TODO: this has the disadvantage of never unclaiming the project from gpalloc. Might need to find a new strategy.
```

Comment:
My idea was to claim a project and store it in a singleton object for the tests to access. But this has the following problems:
- we have no way to release the project
- `BillingFixtures` expects to be mixed in to a `Spec` class, so mixing it into an object feels weird.

Will see how this works, but might land on another solution.